clang/test/SemaSYCL/intel-fpga-local.cpp

```diff
@@ -567,70 +309,13 @@ void diagnostics()
   //expected-error@+1{{merge direction must be 'depth' or 'width'}}
   [[intel::merge("mrg4", "depths")]] unsigned int mrg_invalid_arg[4];
 
-  //Last one is applied and others ignored.
-  //CHECK: VarDecl{{.*}}mrg_mrg
-  //CHECK: SYCLIntelMergeAttr{{.*}}"mrg4" "depth"{{$}}
-  //CHECK: SYCLIntelMergeAttr{{.*}}"mrg5" "width"{{$}}
-  //expected-warning@+2{{attribute 'merge' is already applied}}
```

Comment:
Same here

Reply:
Yes, AST part we are checking inside clang/test/SemaSYCL/intel-fpga-local-ast.cpp file at line 233-238
Diagnostic part we are checking inside clang/test/SemaSYCL/intel-fpga-local.cpp file at line 111-114